.github/workflows/ci-core-reusable.yml

```diff
@@ -60,10 +60,6 @@ jobs:
           ci_run ./bin/zkt
           ci_run zk_supervisor contracts
 
-      # FIXME: enable contract tests once tehy are stable
-      #- name: Contracts unit tests
-      #  run: ci_run yarn l1-contracts test
```

Comment:
we do run those in the main branch:  https://github.com/matter-labs/zksync-era/blob/93bc66f21f9f67a440f06f1c4402e0d687698741/.github/workflows/ci-core-reusable.yml#L63

but I agree that we do not really need them since we already have tests in the era-contracts repo

Reply:
sorry, missed it, I will add it back and let's remove separately